### PR TITLE
Add report about 1st BioNT workshop

### PR DESCRIPTION
**Pull request by bebatut**
Here is a report about the 1st BioNT workshop, that used Galaxy

ping @teresa-m

### CODE REVIEW

File: content/news/2023-09-28-biont-workshop/index.md [hunk 1]
```diff
@@ -0,0 +1,398 @@
+---
+title: '1st BioNT Training workshop - A practical introduction to bioinformatics and RNA-seq using Galaxy'
+tease: "BioNT delivered its first training workshop, A practical introduction to bioinformatics and RNA-seq using Galaxy, from 4th to 8th of September, 2023. This post reports this workshop organisation, the applicants and participants, as well as their feedback about it."
```
Comment (teresa-m) [inline content/news/2023-09-28-biont-workshop/index.md:3]:
```suggestion
tease: "BioNT delivered its first training workshop, A practical introduction to bioinformatics and RNA-seq using Galaxy, from 4th to 8th of September, 2023. This post reports this workshop's organisation, the applicants and participants, as well as their feedback about it."
```

File: content/news/2023-09-28-biont-workshop/index.md [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+
+## Infrastructure for the workshop
+
+The workshop was delivered in a **Zoom webinar format**, with **participants' visibility disabled to preserve their privacy**. Participants were able to see and learn from the trainers. However, **direct interactions (e.g. chat or voice) were not possible** within Zoom during the sessions. All direct interaction took place in **written form via a collaborative document**, where anonymous participation was ensured. To serve the collaborative documents, a HedgeDoc collaborative space was set up by BIOBYTE, and is hosted on their server.
+
+A **Main** collaborative document, setup by the instructors and organisers, was shared with the participants before the session. Each section of the workshop had dedicated _Hands-on_ boxes to report on the task status, ask questions or raise issues. **Helpers engaged and helped participants by answering the questions and issues directly in this document**. This Main document was filled in live during the workshop. Separate boxes to answer questions were used to  improve the participant engagement and as an indirect learning assessment.
```
Comment (teresa-m) [inline content/news/2023-09-28-biont-workshop/index.md:42]:
```suggestion
A **Main** collaborative document, setup by the instructors and organisers, was shared with the participants before the session. Each section of the workshop had dedicated _Hands-on_ boxes to report on the task status, ask questions, or raise issues. **Helpers engaged and helped participants by answering the questions and issues directly in this document**. This Main document was filled in live during the workshop. Separate boxes to answer questions were used to improve participant engagement and as an indirect learning assessment.
```

File: content/news/2023-09-28-biont-workshop/index.md [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+
+To help with the organisation, four HedgeDoc documents were used:
+1. a **[Template](https://biont.biobyte.de/AZptJADHQBusn0t6tpBL5w)** with all instructions and boxes for hands-on, questions, etc;
+2. the **[Main](https://biont.biobyte.de/JvXq44sER_qfh-1J_XIwbg)** document with the information for the participants during the workshop, filled with boxes related to the section covered by the instructor to help with the navigation and cleaned during each break to avoid an overcrowded document;
+3. a **[History](https://biont.biobyte.de/PLOoUieUS4WPbFNQisbz3Q#)** document with everything that needed to be kept and shared with participants after the workshop
```
Comment (teresa-m) [inline content/news/2023-09-28-biont-workshop/index.md:47]:
```suggestion
3. a **[History](https://biont.biobyte.de/s/fDH1UF18o#)** document with everything that needed to be kept and shared with participants after the workshop
```

File: content/news/2023-09-28-biont-workshop/index.md [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+| 3 | Introduction to Transcriptomics | [Slides](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/introduction/slides.html#1) |
+| 3 + 4 | Reference-based RNA-seq | [Tutorial](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/ref-based/tutorial.html) |
+| 5 | Learning about one gene across biological resources and formats | [Slides](https://training.galaxyproject.org/training-material/topics/data-science/tutorials/online-resources-gene/slides.html#1) + [Tutorial](https://training.galaxyproject.org/training-material/topics/data-science/tutorials/online-resources-gene/tutorial.html) |
+| 5 | One protein along the UniProt page | [Tutorial](https://training.galaxyproject.org/training-material/topics/data-science/tutorials/online-resources-protein/tutorial.html) |
+
+## Certificates
+
+Certificates were provided to those participants who explicitly required them after checking for their participation using two criteria: 
+1. they joined some sessions on Zoom and
```
Comment (teresa-m) [inline content/news/2023-09-28-biont-workshop/index.md:67]:
```suggestion
1. they joined at least one session on Zoom or notified that they could not attend live and 
```

**Review by bgruening — APPROVED** [review]
Very nice!

File: content/news/2023-09-28-biont-workshop/index.md [hunk 5]
```diff
@@ -0,0 +1,398 @@
...
+</table>
+
+<img class="img-fluid float-left" src="./figure12.png" style="width:300px;" alt="Barplot displaying answers in the post-workshop survey  to the question: How likely would participants recommend this workshop to a friend or colleague? 0 (0 answers), 1 (0 answers), 2 (0 answers), 3 (0 answers), 4 (0 answers), 5 (0 answers), 6 (1 answers), 7 (0 answers), 8 (5 answers), 9 (0 answers), 10 (22 answers)"/>
+
+The feedback on the post-workshop survey was **positive** and participants were **likely to recommend this workshop to a friend or colleague**.
+
+
+# Conclusion
+
+The first BioNT workshop, a Bioinformatics Introduction workshop, was **successfully** held from 4th to 8th of September 8, 2023.
```
Comment (teresa-m) [inline content/news/2023-09-28-biont-workshop/index.md:394]:
```suggestion
The first BioNT workshop, a Bioinformatics Introduction workshop, was **successfully** held from 4th to 8th of September, 2023. 
```

**Review by teresa-m — APPROVED** [review]
Thanks a lot, @bebatut. The blog post is a very nice read.

**Comment by bgruening** [comment]
Thanks @bebatut

**Pull request merged**